### PR TITLE
Enable to view conference day tab even if font size is large

### PR DESCRIPTION
**Pull request by ked4ma**
## Issue
- no issue

## Overview (Required)
- As attached screenshots, conference day tab is not shown correctly if font size is changed.
- Font was not set
- layout was little bit far from figma's
  
- **If this pr is not fit to the app, please close** :bow:

## Links
- https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI?node-id=54849-12900&t=Kr9KWAn23JSTK8Y7-0

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 
 |

### CODE REVIEW

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridTab.kt [hunk 1]
```diff
@@ -32,17 +34,21 @@ fun TimetableDayTab(
     onDaySelected: (day: DroidKaigi2024Day) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    val density = LocalDensity.current
+    val tabWidth = with(density) {
+        // calculate width from sp
+        // NOTE: 6 is a magic number here
+        ((16.sp * 6) * fontScale).toDp()
+    }
```
Comment (ked4ma) [inline feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridTab.kt:42]: `TabRow` try to use max width as much as possible, so we need to specify fixed width.
For this, calculating the width (`dp`) from `sp` using `density`.

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridTab.kt [hunk 2]
```diff
@@ -90,8 +92,9 @@ private fun FloorText(
     isSelected: Boolean,
 ) {
     Text(
+        modifier = Modifier.padding(bottom = 4.dp),
         text = text,
-        fontWeight = FontWeight.W400,
+        style = MaterialTheme.typography.titleMedium,
```
Comment (ked4ma) [inline feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridTab.kt:97]: attached font that is used for other components.

**Comment by takahirom** [comment]
@r4wxii 
I wanted to let you know that there are some changes to the tab. I think we have some room for improvement in the tab design. So, I'll merge this pull request.
https://github.com/DroidKaigi/conference-app-2024/issues/704

**Review by takahirom — APPROVED** [review]
Looks great! Thank you for your contribution!

**Pull request merged**